### DISCUSSION TITLE
MOD没有重复却显示重复

### DISCUSSION THREAD

**Discussion by mjyf** [Meloong-Git/PCL#4660] [提问] [answered]
版本：1.20.1
模组加载器：Forge
MOD：Xaero的世界地图，Xaero的小地图，机械动力，连锁破坏，铷，玉，Architecyury API,Embeddium,FTB Library,Oculus
错误报告：[错误报告-2024-8-30_19.26.23.zip](https://github.com/user-attachments/files/16823420/-2024-8-30_19.26.23.zip)
状况：
![image](https://github.com/user-attachments/assets/6731cce1-ac32-4805-9c81-9db256a1531e)
![image](https://github.com/user-attachments/assets/d53819f8-7dd2-45ed-8104-cc834af87a25)
![image](https://github.com/user-attachments/assets/f90181f4-282c-4746-8fed-bed2c91a8edc)

**Comment by 3gf8jv4dv** [comment, marked as answer]
Choose between Rubidium and Embeddium. Do not install both at the same time.
Rubidium 和 Embeddium 二选一，不要同时安装。

**Reply by mjyf** [reply]
谢谢
